_docs_use-cases/use-case-transport-upload-own-file.md

```diff
@@ -57,7 +57,7 @@ This use case uses a given file outside of enmeshed, encrypts and uploads it to
 
 - An encrypted File is stored on the Backbone and can be shared by its references.
 - Only the metadata of the File is stored locally.
-- If `expiresAt` wasn't speficied, the expiration date of the File will be set to a default value that corresponds to the 31st of December, 9999.
+- If `expiresAt` wasn't speficied, the expiration date of the File will be set to a default value that corresponds to zero o'clock of the 31st of December, 9999.
```

Comment:
- Write "specified" instead of "speficied".
- Write "...zero o'clock **on** the 31st of December, 9999." instead of "...zero o'clock of the 31st of December, 9999.".

Reply:
- (fremdschämende Frau)
- According to my research both are fine. I think the slight difference is that the former translates to "Null Uhr am 31. Dezember, 9999" and the latter "Null Uhr des 31. Dezembers, 9999". I prefer the latter.